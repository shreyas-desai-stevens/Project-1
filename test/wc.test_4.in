
s
pp
mcn
vphc
pcrwd
jltwst
bdccpty
xquepesz
bmjgpprfo

z
bh
asj
ucpm
mhdve
fjezju
lwmbfat
qtlagcpg
sujyaajgt

n
ia
sql
navr
xqcxm
zclvdo
tojepgw
qtnrvzef
qbbgysnae

z
tt
ebl
dbhn
kcdku
gyaotu
cupqqmp
fzgbhuwz
evqvsnkwl

g
cc
evo
fyfx
weotg
mssqyv
oiovxkw
jukvruyr
pzicdoclk

t
cb
gkb
zkpp
pdztd
rhgfto
hrokqvr
xqdgdcbn
jbtpajdek

x
cb
ypu
zktc
pjhsy
lhaslv
hfaopqm
rtiwlvqv
nelpvmgiv

i
jy
vrz
zvrw
jwvpj
gvezyy
rddlbee
nixygzlb
taycarqjr

s
yn
drl
wplz
okskq
lvlcfa
ugovfis
ciujnobo
vehxvundu

p
wc
rfl
qbbf
lqvrd
pcsyzl
gqldtkz
ljrsscpu
zctnrsxmu
