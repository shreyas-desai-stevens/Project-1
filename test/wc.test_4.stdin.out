    100    90    550 